### DISCUSSION TITLE
unregister import chasers

### DISCUSSION THREAD

**Discussion by jhultgre** [Autodesk/maya-usd#1900] [Q&A] [answered]
Is there a way to unregister an import chaser?
I'm working on my code and reregisterd my class with some updates with the same name, but it seems to be running the old code still.
Or is there some other way to reload the chaser when I'm working on it?
Do I just have to register it with a random name every time I run my code so it uses the latest?

**Comment by boudrey** [comment, marked as answer]
There is no way for now to unregister a chaser.
As a workaround generating a random chaser name is a good idea. You could also call a global functions from your import chaser if you can restrict your changes to the content of those functions. Otherwise you need to restart Maya.
